pom.xml

```diff
@@ -67,11 +67,11 @@
     <mockwebserver.version>3.5.0</mockwebserver.version>
 
     <!-- Production -->
-    <cxf.version>3.3.2</cxf.version>
+    <cxf.version>3.3.5</cxf.version>
     <cxf.java2ws.plugin.version>${cxf.version}</cxf.java2ws.plugin.version>
-    <httpcomponents.httpcore.version>4.4.11</httpcomponents.httpcore.version>
+    <httpcomponents.httpcore.version>4.4.12</httpcomponents.httpcore.version>
     <httpcomponents.httpasyncclient.version>4.1.4</httpcomponents.httpasyncclient.version>
-    <httpcomponents.httpclient.version>4.5.8</httpcomponents.httpclient.version>
+    <httpcomponents.httpclient.version>4.5.10</httpcomponents.httpclient.version>
```

Comment:
https://github.com/apache/brooklyn-server/pull/1096/files is showing 4.5.12

Reply:
As above: 4.5.10 is right.